resources/production/config-example.toml

```diff
@@ -6,8 +6,16 @@
 # If set, use this hash as a trust anchor when joining an existing network.
 #trusted_hash = 'HEX-FORMATTED BLOCK HASH'
 
-# Whether to synchronize all data back to genesis, or just [deploys.max_ttl] (chainspec setting) worth of data.
-sync_to_genesis = true
+# Historical sync behavior for this node. Options are:
+#  'genesis'  (node will attempt to acquire all block data back to genesis)
+#  'ttl'      (node will attempt to acquire all block data to comply with time to live enforcement)
+#  'nosync'   (node will only acquire blocks moving forward)
```

Comment:
@EdHastingsCasperAssociation so a "nosync" node will still acquire blocks etc in the storage.lmdb ? A small improvement would be to clean the storage lmdb on restart to regain previous blocks acquired. (The improvement will be really small tho')